sdk/tests/client/common/mod.rs

```diff
@@ -8,9 +8,9 @@ mod constants;
 //     api::GetAddressesOptions, constants::SHIMMER_COIN_TYPE, node_api::indexer::query_parameters::QueryParameter,
 //     request_funds_from_faucet, secret::SecretManager, Client, Result,
 // };
-use iota_sdk::client::Client;
+use iota_sdk::client::{Client, Result};
 
-pub use self::constants::{FAUCET_URL, NODE_LOCAL};
+pub use self::constants::*;
```

Comment:
We usually avoid wildcards, I know it's tests so feel free to just resolve but I would personally revert for consistency

Reply:
okay

Reply:
wen

Reply:
on it

Reply:
@thibault-martinez Done